### DISCUSSION TITLE
Website

### DISCUSSION THREAD

**Discussion by ghost** [techno-tim/techno-tim.github.io#90] [Questions] [answered]
My website is not working

https://github.com/techno-tim/techno-tim.github.io/blob/f693b6a38690d44122641f23b65158cc91e95a43/feed.xml#L51

The repository can't find `no-linenos.html` in `_includes` but it's working for your repository.

My repository is linked [here](https://github.com/codedsimon/codedsimon.github.io).

**Comment by timothystewart6** [comment]
hey, not sure!

**Comment by ghost** [comment]
what do you mean

**Comment by timothystewart6** [comment]
I am not sure what the issue is with your website

**Reply by ghost** [reply]
Maybe I haven't wrote it correctly;

My repository is missing no-linenos.html in the _includes folder @techno-Tim.

Your website is working without a no-linenos.html file and github pages is working but the problem with my one is that in feed.html there is a no-linenos.html file listed and it checked the _includes folder but couldn't find no-linenos.html.

Check that repo out [here](https://github.com/codedsimon/codedsimon.github.io) and inform me if you see any mistakes cause I'm already looking.

_**I think the main mistake is on line 51**_

**Comment by timothystewart6** [comment, marked as answer]
If you're not getting any responses here in the discussions tab, feel free to join our Discord for questions! http://l.technotim.live/discord
